faker.go

```diff
@@ -168,14 +168,18 @@ func getValue(t reflect.Type) (reflect.Value, error) {
 					continue // to avoid panic to set on unexported field in struct
 				}
 				tag := t.Field(i).Tag.Get(tagName)
-				if tag == "" {
+
+				switch tag {
+				case "":
 					val, err := getValue(v.Field(i).Type())
 					if err != nil {
 						return reflect.Value{}, err
 					}
 					val = val.Convert(v.Field(i).Type())
 					v.Field(i).Set(val)
-				} else {
+				case "skip":
```

Comment:
thank you for the PR. 
We forgot to add this. 
IMHO, I think it's better to use dash (`-`) instead of `skip` like what JSON tag's do if we want to ignore some field. 

```go
type Sample struct {
 ID string `json:"-"`

}

```

Reply:
Good idea. :) smaller and simpler. Done.